developer-guide.md

```diff
@@ -105,11 +105,11 @@ Currently go to the Octopus Team City instance and navigate to: `Build Integrati
 Select `Run` and fill in the new version in the dialog:
 
 ![run custom build](./developer-guide-images/run-custom-build.png)
-
-**Please note**: Before trigger the build you need to ensure that
-1. The commit isn't tagged with the version
-1. The version number in the `<version>` tag in [pom.xml](https://github.com/OctopusDeploy/octopus-jenkins-plugin/blob/master/pom.xml) is suffixed with `-SNAPSHOT`.
-1. If the Teamcity build fails, it might have already committed changes to the git repository, you will need to manually revert changes before trying again.
+
+**Please note**: Before trigger the build you need to ensure that
+1. The commit isn't tagged with the version
+1. The version number in the `<version>` tag in [pom.xml](https://github.com/OctopusDeploy/octopus-jenkins-plugin/blob/master/pom.xml) is suffixed with `-SNAPSHOT`.
+1. If the Teamcity build fails, it might have already committed changes to the git repository, you will need to manually revert changes before trying again.
```

Comment:
the teamcity build commits back to the git repo? that sounds ... interesting.

Reply:
Yeah I don't know what's going on here.. some merge issues unrelated to my change